src/main/java/uk/gov/hmcts/reform/wataskconfigurationapi/auth/role/entities/request/QueryRequest.java

```diff
@@ -32,6 +33,8 @@
     private final List<RoleCategory> roleCategory;
     private final Map<String, List<String>> attributes;
     private final List<String> authorisations;
+    private final List<String> hasAttributes;
+    private final Boolean readOnly;
```

Comment:
Maybe consider a native boolean?  Could this be null or should we default to false?

Reply:
These are the Am-role assignment entities, however, we could rename them within our app